extend/generic-extractor/registration.md

```diff
@@ -329,21 +329,21 @@ Save the file into a separate `template.json` file. The template file therefore
                     "type": {
                         "attr": "userType"
                     }
-                }                    
+                }
             },
             {
                 "endpoint": "orders",
                 "dataType": "orders",
                 "params": {
                     "accountId": {
                         "attr": "accountId"
-                    }                        
+                    }
                 }
             }
         ]
     }
 }
 {% endhighlight %}
 
-Create as many `template.json` files as you wish. However, all of them need to share the same `api.json` 
-configuration. When you want to register your component, attach the `api.json` and all `template.json` files. 
+Create as many `template.json` files as you wish. However, all of them need to share the same `api.json`
+configuration. When you want to register your component, attach the `api.json` and all `template.json` files.

```

Comment:
co jim říct, ať pošlou PR na repo na githubu? Bych ten repozitář mohl i nějak zdokumentovat, aby jim to pomohlo...

Reply:
PR jsem dopsal, ale co bys k tomu jeste chtel dal psat? prijde mi, ze je to celkem complete https://developers.keboola.com/extend/generic-extractor/registration/#example

Reply:
Asi bych jenom dal zpětnej link z https://github.com/keboola/kbc-ui-templates/blob/master/README.md na https://developers.keboola.com/extend/generic-extractor/registration/#example :-)